Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -56,6 +73,7 @@ public PowerShellKernel()
                     $"{psJupyterModulePath}{Path.PathSeparator}{psModulePath}");
 
                 RegisterForDisposal(pwsh);
+
```

Comment:
Can you remove this line? This will make it easier for rebasing my host branch later :)

Reply:
done.